stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/channels/header/ChannelListHeader.kt

```diff
@@ -134,14 +135,13 @@ internal fun DefaultChannelHeaderLeadingContent(
     currentUser: User?,
     onAvatarClick: (User?) -> Unit,
 ) {
-    val size = Modifier.size(40.dp)
+    val size = Modifier.size(ChatTheme.dimens.channelAvatarSize)
 
     if (currentUser != null) {
-        UserAvatar(
+        Avatar(
```

Comment:
Why are we switching from `UserAvatar` to `Avatar`?
Shouldn't we use the factory here?

Reply:
I replaced all calls from `UserAvatar` to `Avatar` when `showOnlineIndicator = false`
That means it's not meant to show the online indicator at any case.

Reply:
The `Avatar` factory is not done yet. As I mentioned, it's a WIP. PR is a draft yet